_posts/2024-08-07-taming-apache-kafka-4.0.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title:  "Taming Apache Kafka 4.0"
+date: 2024-08-07
+author: jakub_scholz
+---
+
+Apache Kafka 4.0 release is just around the corner.
+It will be a very significant release for Apache Kafka.
+But it will also have a huge impact on Strimzi.
+Therefore in this blog post, we will look closer at what changes will Apache Kafka 4.0 bring, when is it currently planned to be released and how will it be supported by Strimzi.
```

Comment:
```suggestion
Therefore in this blog post, we will look closer at what changes Apache Kafka 4.0 will bring, when is it currently planned to be released, and how it will be supported by Strimzi.
```